src/components/game/stats/index.jsx

```diff
@@ -45,6 +48,43 @@ class Stats extends React.Component {
         <p>
           Vertices: { render.vertices }
         </p>
+
+        <div className="divider"></div>
+
+        <h2>Map Doodads</h2>
+        <div className="divider"></div>
+        <p>
+          Loading: { map ? map.doodadManager.entriesPendingLoad.size : 0 }
+        </p>
+        <p>
+          Loaded: { map ? map.doodadManager.doodads.size : 0 }
+        </p>
+        <p>
+          Animated: { map ? map.doodadManager.animatedDoodads.size : 0 }
+        </p>
+
+        <div className="divider"></div>
+
+        <h2>WMOs</h2>
+        <div className="divider"></div>
+        <p>
+          Loading Roots: { map ? map.wmoManager.entriesPendingLoad.size : 0 }
+        </p>
+        <p>
+          Loaded Roots: { map ? map.wmoManager.wmos.size : 0 }
+        </p>
+        <p>
+          Loading Groups: { map ? map.wmoManager.groupsPendingLoadCount : 0 }
+        </p>
+        <p>
+          Loaded Groups: { map ? map.wmoManager.groupCount : 0 }
+        </p>
+        <p>
+          Loading Doodads: { map ? map.wmoManager.doodadsPendingLoadCount : 0 }
+        </p>
+        <p>
+          Loaded Doodads: { map ? map.wmoManager.doodadCount : 0 }
+        </p>
```

Comment:
Nice, like this a lot!

Would it be an idea to drop this content entirely if there's no map? Something like:

```
get mapStats() {
  return <div><h2>Doodads</h2><p>Loaded: { map.doodadManager... }</p> etc.</div>;
}

render() {
  return <stats>Regular stuff { map && this.mapStats }</stats>;
}
```